lua/nvim-lsp-installer/servers/gopls.lua

```diff
@@ -0,0 +1,31 @@
+local server = require('nvim-lsp-installer.server')
+
+local root_dir = server.get_server_root_path('go')
+
+local install_cmd = [=[
+if ! command -v go &> /dev/null;
+then 
+  echo "Please install the Go CLI before installing gopls.";
+  echo "refer to https://golang.org/doc/install";
+  exit 1;
+fi
+
+GO111MODULE=on go get golang.org/x/tools/gopls@latest;
```

Comment:
Not super familiar with go - but would it be possible to install the package inside the current working dir?

I'm thinking something like:

```sh
GO111MODULE=on GOPATH="$PWD" go get golang.org/x/tools/gopls@latest
```

The reason for this is to avoid polluting the system and have the installation confined within the LSP server's root_dir.

Then the `cmd` below would need to become something like:

```lua
cmd = { root_dir .. "/bin/gopls" }
```

What do you think?

Reply:
After some searching, I came up with https://github.com/ecmma/nvim-lsp-installer/commit/747739412f110c701685605e7af12bdb7b689d76. 

However, I noticed that the server doesn't start unless I close and reopen the *.go file, while the normal behaviour is directly start the server, and right now I don't know what causes this (but I don't think it's related to this change.)

Reply:
> However, I noticed that the server doesn't start unless I close and reopen the *.go file, while the normal behaviour is directly start the server, and right now I don't know what causes this (but I don't think it's related to this change.)

Hm sounds a bit odd but yeah probably not related to this PR. Could it be that the buffer doesn't connect itself to the LSP server, rather than the server not starting? Maybe something's up with your `on_attach` function 🤔